src/goto-programs/process_goto_program.h

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************\
+
+Module: Process a Goto Program
+
+Author: Martin Brain, martin.brain@cs.ox.ac.uk
+
+\*******************************************************************/
+
+/// \file
+/// Common processing and simplification of goto_programts.
+/// This includes linking libraries and removing a number of
+/// more complex types (vectors, complex, etc.) and constructs
+/// (returns, function pointers, etc.).
+/// This is can be used after initialize_goto_model but before
+/// analysis.  It is not madatory but is used by most tools.
```

Comment:
Nit picking: s/madatory/mandatory/
But also: could/should this comment be moved towards the function documentation rather than the file's documentation?

Reply:
Fixed.